### PR TITLE
Restyling of content sections (bootstrap)

### PR DESCRIPTION
**Pull request by MeganCFord**
I believe this is how this PR needs to go. it's restyling of the content sections in bootstrap, and it's built off the navbar PR. No content changes- the website should look almost exactly the same, but be more stable across screen-sizes.

I also removed the gallery section, I assume you still wanted that.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,103 +84,74 @@
...
+          <p>HoneyCo rental homes are located in multigenerational neighborhoods. These homes offer the resident a
+            maintenance-free lifestyle, supported by smart home technology and connected to community. Make the most of
+            your unRetirement in a HoneyCo Home.</p>
+          <a class="btn-yellow" href="#connect" data-scroll>Learn More</a>
+        </div>
+      </div>
     </div>
-  </div>
 
-  <div id="gallery" class="gallery">
```
Comment (MeganCFord) [inline index.html]: I assume you still wanted to remove the gallery section here.

File: scss/style.scss [hunk 2]
```diff
@@ -178,136 +185,76 @@ h3 {
...
-          img {
-            transform: translate(-100%, -50%);
-          }
-        }
-      }
-    }
+.security {
+  background: no-repeat url("../img/security.jpg") top center;
+  background-size: cover;
+  margin: 0 -15px;
```
Comment (MeganCFord) [inline scss/style.scss]: -15px to offset container-fluid padding, for full-width background image.

File: index.html [hunk 3]
```diff
@@ -104,10 +104,11 @@
     <div id="security" class="row split-section security">
       <div class="col-xs-12 col-sm-9 col-md-6 text">
         <div class="inner text-xs-center text-sm-left">
-          <h3>The Internet of caring things<span class="pour-some-trademark-on-me">.<span class="trademark">TM</span></span></h3>
+          <h3>The Internet of caring things&trade;</h3>
```
Comment (MeganCFord) [inline index.html]: this would ensure the trademark symbol is the same monteserrat font?

Comment (zachmccormick) [inline index.html]: Eh I'm not too concerned.  I also think it's going to go soon

**Pull request merged**